sdk/src/wallet/account/update.rs

```diff
@@ -276,9 +275,8 @@ impl Account {
 
     // Should only be called from the Wallet so all accounts are on the same state
     // Will update the addresses with a possible new Bech32 HRP and clear the inaccessible_incoming_transactions.
-    pub(crate) async fn update_account_with_new_client(&mut self, client: Client) -> crate::wallet::Result<()> {
-        self.client = client;
-        let bech32_hrp = self.client.get_bech32_hrp().await?;
+    pub(crate) async fn update_account_with_new_client(&mut self) -> crate::wallet::Result<()> {
```

Comment:
Just checking, the name suggest a new client, I don't exactly know where it was coming from. Is it actually right to update with the wallet client?

Reply:
`update_account_with_updated_client` :)

Reply:
Maybe `update_account_with_bech32_hrp` is better, since that seems to be the focus?

Reply:
`with_` sounds like it's a parameter, maybe just `update_account_bech32_hrp`